openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/AdvanceSearch.spec.js

```diff
@@ -116,7 +114,7 @@ describe('Advance search should work properly for all fields', () => {
   });
 });
 
-describe('Advance search should work properly for Add Group functionality', () => {
+describe.skip('Advance search should work properly for Add Group functionality', () => {
```

Comment:
Same, can we add comments with the reason behind skipping?

Reply:
Thanks for pointing this out but its already discussed, for time being we need to skip this test due to the ES issue